backend/vaa-strapi/src/extensions/users-permissions/controllers/candidate.ts

```diff
@@ -87,9 +87,16 @@ module.exports = {
 
     const pluginStore = await strapi.store({type: 'plugin', name: 'users-permissions'});
     const settings = await pluginStore.get({key: 'advanced'});
-    const role = await strapi
-      .query('plugin::users-permissions.role')
-      .findOne({where: {type: settings.default_role}});
+
+    const role = await strapi.query('plugin::users-permissions.role').findOne({
+      where: {
+        type: 'authenticated'
```

Comment:
TODO: Fix this, old way was giving property does not exist TypeScript validation errors